sphinx/builders/html.py

```diff
@@ -581,6 +580,14 @@ def finish(self):
         # dump the search index
         self.handle_finish()
 
+    def interrupt(self):
+        # type: () -> None
+        self.interrupt_tasks.add_task(self.copy_image_files)
+        self.interrupt_tasks.add_task(self.copy_download_files)
+        # not only copying, generating.
+        # self.interrupt_tasks.add_task(self.copy_static_files)
```

Comment:
We should call `copy_static_files` also if you want to fix #1649.
I know this resolved the problem of #1649 itself. But this is not consistent behavior. So the another issue will be posted.